src/System.Collections/src/System/Collections/Generic/SortedList.cs

```diff
@@ -443,19 +443,19 @@ public bool ContainsValue(TValue value)
         }
 
         // Copies the values in this SortedList to an array.
-        void ICollection<KeyValuePair<TKey, TValue>>.CopyTo(KeyValuePair<TKey, TValue>[] array, int arrayIndex)
+        void ICollection<KeyValuePair<TKey, TValue>>.CopyTo(KeyValuePair<TKey, TValue>[] array, int index)
```

Comment:
Why are you changing the parameter name?  It's named ```arrayIndex``` in the interface itself.  Unless there's a really good reason to change it, we should leave it as is.